guides/source/users/promotions/promotion-actions.html.md

```diff
@@ -25,8 +25,13 @@ following steps:
    of many types of discount depending on the promotion calculator being used.
    (For example, the amount could be a flat percentage or a flexible rate.)
 
-Note that if you choose the promotion action type **Free Shipping** that steps
-3. and 4. are not applicable.
+### Notes
+1. If you choose the promotion action type **Free Shipping**, then steps
+**3** and **4** are not applicable.
+2. A warning message is logged if the following conditions are true:
```

Comment:
I'd use a more admin-friendly language here. They are not supposed to be technical people and maybe a more conversational tone of voice is more appropriate here.

Reply:
How about:

`2. You can specify a coupon code to your promotion, but your customers
will not be able to apply it if no actions are added to them.`